examples/jsm/node-editor/NodeEditor.js

```diff
@@ -359,6 +387,38 @@ export class NodeEditor extends EventDispatcher {
 
 	}
 
+	set preview( value ) {
```

Comment:
What if the setter is run with the same value as of `this._preview`? I think there should be a check at the start of it.